.travis.yml

```diff
@@ -2,7 +2,8 @@ sudo: false
 dist: trusty
 language: ruby
 rvm:
-- 2.4.1
-- 2.3.4
+- 2.5
+- 2.4
+- 2.3
```

Comment:
Unfortunately this will not use the latest point releases because of the old version of RVM that is used on Travis CI. It only resolves `2.5` to `2.5.1` because it does not know anything of `2.5` at all and has to download and compile it. The versions `2.4` and `2.3` are actually resolved to to `2.4.1` and `2.3.4` respectively instead of `2.4.4` and `2.3.7`. I've not found any reliable why beside  manually updating all versions, but thanks for remembering me.

Reply:
Wow. I had no idea.